ts/Core/Series/Series.ts

```diff
@@ -432,8 +432,15 @@ class Series {
         const options = series.options,
             visible = options.visible !== false;
 
+        /**
+         * All child series belonging to the current series.
```

Comment:
Should be more elaborated:

All child series that are linked to the current series through the [linkedTo](https://api.highcharts.com/highcharts/series.line.linkedTo) option.